### ISSUE TITLE
Using custom port for  login link.

### ISSUE THREAD

**Issue by lordcf** [cloudfoundry/uaa#856]
Hi Team, I m working on a PoC in AWS china, beijing region, as per china gov rule, Currently I cannot access CF on standard ports (80,443,8080). 
To overcome this I  put my router VM  behind AWS ELB, Now  my ELB listen on port 3443(TCP) and forward it to router on  443(TCP).

I'm able to set cf api with command:
`cf api api.sys.mydomain.com:3443 --skip-ssl-validation`

But CF login fails with error
`Error performing request: Get https://login.sys.mydomain.com/login: dial tcp 52.xx.xxxx.xx:443: i/o timeout`
Instead of using custom <domain:port> (3443) login request is sent to standard https domain 

What is the source for generating this login url, I tried by updating all the url  related to login url in Manifest with additionof  `:3443`. Doesn't work.

Is there any specific property I can set this port/functionality.

```
cf curl /v2/info
{
   "name": "",
   "build": "",
   "support": "http://support.cloudfoundry.com",
   "version": 0,
   "description": "",
   "authorization_endpoint": "https://login.sys.mydomain.com",
   "token_endpoint": "https://uaa.sys.mydomain.com:3443",
   "min_cli_version": null,
   "min_recommended_cli_version": null,
   "api_version": "2.97.0",
   "app_ssh_endpoint": "ssh.sys.mydomain.com:2222",
   "app_ssh_host_key_fingerprint": "25:ad:xx:xx:51:e1:48:xx:xx:d9:77:18:xx:xx:a2:e2",
   "app_ssh_oauth_client": "ssh-proxy",
   "doppler_logging_endpoint": "wss://doppler.sys.mydomain.com:443"
}
```

**Comment by DennisDenuto** [comment]
Hello @lordcf Thanks for creating this issue. I am going to make an assumption that you have deployed your `cf` with bosh.

In order to change the port that is returned by the `/info` endpoint, you will need to configure the [uaa.url property](https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/spec#L138) to -> `https://login.sys.mydomain.com:3443`.

Let us know how it goes!

**Comment by lordcf** [comment]
sorry for delayed reply.
As per your suggestion  I made change in my manifest, I updated ['properties.login.url'](https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/spec#L1194) instead of  'uaa.url'](https://github.com/cloudfoundry/uaa-release/blob/develop/jobs/uaa/spec#L138).

Now I can see intended  `"authorization_endpoint":"https://login.sys.cfcn.portal-dev.bentleymotors.com:3443"` 

but cf login fails with 404. On browser also I get
`404 Not Found: Requested route ('login.sys.mydomain.com:3443') does not exist.`

What is the difference between, uaa.url vs login.url.   how it will affect on CF if I change uaa.url to login.xxxx.com
